Snowflake.Data.Tests/SFConnectionIT.cs

```diff
@@ -744,6 +746,31 @@ public void TestSSOConnectionWithUser()
             }
         }
 
+        [Test]
+        [Ignore("This test requires manual interaction and therefore cannot be run in CI")]

```

Comment:
are there any additional actions or pre steps necessary to run this test? e.g. I have to set up external okta account?

Reply:
And where is it described how to do it?

Reply:
In internal doc